com.rawtherapee.RawTherapee.yaml

```diff
@@ -186,9 +186,10 @@ modules:
     config-opts:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
       - -DCACHE_NAME_SUFFIX=""
+      - -DCMAKE_CXX_FLAGS=-O3
     sources:
       - type: archive
-        url: https://github.com/Beep6581/RawTherapee/releases/download/5.9/rawtherapee-5.9.tar.xz
+        url: https://rawtherapee.com/shared/source/rawtherapee-5.9.tar.xz
```

Comment:
there is no point into that change.

Reply:
Yes, I thought that too. However if the folks from rawtherapee took the time to advise packagers against downloading tarballs from github, there should be a good reason for that. Since the hash is the same, following their counsel is quite easy, so I simply did it.

Reply:
You just misunderstand what they wrote. It's the "auto generated" tarball they advise against. Which is just a tarbal of the tree without any of the submodules. If the hash is the same then the content is the same.

Reply:
Ah ok, I understand now. Let me undo it then.